tictactoe.js

```diff
@@ -0,0 +1,136 @@
+
+player= "X";
+space = document.getElementsByClassName("space");
+
+
+// var boardCheck;
+
+// var cellValue = function cells(key) {
+//     switch(key) {
+//         case 'a': return "";
+//         case 'b': return "";
+//         case 'c': return "";
+//         case 'd': return "";
+//         case 'e': return "";
+//         case 'f': return "";
+//         case 'g': return "";
+//         case 'h': return "";
+//         case 'i': return "";
+//         default : return "";
+//     }
+// }
+
+
+//-------------------------------------------------------------------
+function clickSpace() {
+    if (this.id == "a") {
+        if(document.getElementById("a").innerHTML == ""){ 
+           document.getElementById("a").innerHTML == player;
+           changeTurn();
+         }
+    } else if (this.id == "b") {
+        if(document.getElementById("b").innerHTML == ""){ 
+           document.getElementById("b").innerHTML = player;
+           changeTurn();
+         }
+    } else if (this.id == "c") {
+        if(document.getElementById("c").innerHTML == ""){ 
+           document.getElementById("c").innerHTML = player;
+           changeTurn();
+         }
+
+    } else if (this.id == "d") {
+      if(document.getElementById("d").innerHTML == ""){ 
+           document.getElementById("d").innerHTML = player;
+           changeTurn();
+         }
+    } else if (this.id == "e") {
+        if(document.getElementById("e").innerHTML == ""){ 
+           document.getElementById("e").innerHTML = player;
+           changeTurn();
+         }
+    } else if (this.id == "f") {
+        if(document.getElementById("f").innerHTML == ""){ 
+           document.getElementById("f").innerHTML = player;
+           changeTurn();
+         }
+
+    } else if (this.id == "g") {
+        if(document.getElementById("g").innerHTML == ""){ 
+           document.getElementById("g").innerHTML = player;
+           changeTurn();
+           }
+    } else if (this.id == "h") {
+        if(document.getElementById("h").innerHTML == ""){ 
+           document.getElementById("h").innerHTML = player;
+           changeTurn();
+         }
+    } else if (this.id == "i") {
+        if(document.getElementById("i").innerHTML == ""){ 
+           document.getElementById("i").innerHTML = player;
+           changeTurn();
+    }
+}
+    
+}
+
+document.getElementById("a").onclick = clickSpace;
```

Comment:
After testing, the following javascript should give you the desired result:
```
document.getElementById("a").addEventListener('click', clickSpace);
```
To clean this up even further, you can loop through these elements so that you don't have to write this line of code more than once.